settings.cfg.sh

```diff
@@ -8,6 +8,9 @@ LSSTSW_BUILD_DIR=${LSSTSW_BUILD_DIR:-${LSSTSW}/build}
 # lsstswBuild.sh
 # options passed to ./create_xlinkdocs.sh as cli options
 DOC_PUSH_PATH=${DOC_PUSH_PATH:-"$HOME/public_html/doxygen"}
+# lsstcam
+LSSTCAM_BUCKET=${LSSTCAM_BUCKET:-'rubin-ci-lsst/testdata_ci_lsstcam_m49'}
```

Comment:
I'm a little worried this is insufficiently general for when we have multiple test datasets using this mechanism.  AP is already creating something similar (although for internal USDF use only at this time, it may come to Jenkins in the future).
Having a list of directories to copy out of the single bucket might be better.